docker/demo_server/docker-compose.yml

```diff
@@ -13,15 +13,23 @@ services:
       WEB_HOST: '${WEB_URL}'
       JWT_SECRET: '${JWT_SECRET}'
     volumes:
-      - ./conforma_on_port_${PORT_APP}/postgres:/var/lib/postgresql/12/main/
-      - ./conforma_on_port_${PORT_APP}/snapshots:/usr/src/application-manager-server/build/database/_snapshots
-      - ./conforma_on_port_${PORT_APP}/files:/usr/src/application-manager-server/build/files
-      - ./conforma_on_port_${PORT_APP}/localisation:/usr/src/application-manager-server/build/localisation
-      - ./conforma_on_port_${PORT_APP}/preferences:/usr/src/application-manager-server/build/preferences
+      - postgres:/var/lib/postgresql/12/main/
+      - snapshots:/usr/src/application-manager-server/build/database/_snapshots
+      - files:/usr/src/application-manager-server/build/files
+      - localisation:/usr/src/application-manager-server/build/localisation
+      - preferences:/usr/src/application-manager-server/build/preferences
```

Comment:
These are the new "volumes". Docker automatically adds the instance name to them, so don't need the port variable any more.

The volumes are stored in `/var/lib/docker/volumes` in the container, but I've made a symlink to this folder in the usual `/demo_server` folder where the current volumes are kept (on the demo server)

Reply:
Hmmm is this documented somewhere?

Reply:
I plan to update the docs, was waiting for your docs PR to be in. Will do soon.